Userland/Libraries/LibWeb/CSS/CSSPropertyRule.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024, Alex Ungurianu <alex@ungurianu.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/CSSPropertyRulePrototype.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/CSS/CSSPropertyRule.h>
+#include <LibWeb/CSS/Parser/Parser.h>
```

Comment:
We don't need to include the Parser here do we?

Reply:
Replaced with the correct source of the serializer functions, which is `CSS/Serializer`